plugins/modules/aci_rest.py

```diff
@@ -284,9 +288,11 @@
     HAS_YAML = False
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
 from ansible.module_utils._text import to_text
 
+ANNOTATION_UNSUPPORTED = ["tagTag"]
```

Comment:
I think this list should be expanded with all configurable classes that do not have annotation property, and placed in the constants file

Reply:
This is a good idea. However, after we did some analysis we found 830 classes that don't use annotation properties.
Our options are:
1. We could have a big list in the constants file. (Fastest)
2. Include a meta data file and read annotation supported classes from there. (Slower)
3. Have a retry mechanism that checks for errors relating to annotation (More calls to Apic, possibly slowest)
I think 1 is fine but it will be a big list.
@lhercot any thoughts?

Reply:
We can do 1. with a small script that runs as part of the release_pr process to update from the published documentation so we never have to update it manually? In that case might be easier to have a separate file for just that variable but not sure.

Reply:
Okay that's a great idea. I'll have to figure out how to implement it, so for now I'll add the scripted release_pr process to the backlog and do option one temporarily.

Reply:
Temporary list of classes implemented and scripted process added to backlog.